experiments/integrated/ozark/ozark.jl

```diff
@@ -268,6 +268,20 @@ function diurnal_avg(series)
         [mean([daily_data[i][j] for i in 1:num_days]) for j in 1:daily_points]
     return daily_avgs
 end
+
+# This function scales 2 data series to the same length and matches their
+# indices such that they may be used to compute GOF stats
+function match_indices(model, data)
```

Comment:
could we add this function to a new julia file in experiments/integrated? maybe "statistical_tools.jl" or something. then we can use this in all scripts without duplicating. not sure if your current work on making a single script would obviate the need for this already, though!

Reply:
I think my current work making a single script even exacerbates this need! I am going to add some data checking tools and things like this that I am thinking of moving to a separate file.